triton_backend/inflight_batcher_llm/scripts/build.sh

```
@@ -53,7 +53,7 @@ fi
 
 # TODO: Remove specifying Triton version after cmake version is upgraded to 3.31.8
 # Get TRITON_SHORT_TAG from docker/Dockerfile.multi
-LLM_ROOT="$(dirname $0)/../../../.."
+LLM_ROOT="$(dirname $0)/../../.."
 TRITON_SHORT_TAG=$("$LLM_ROOT/jenkins/scripts/get_triton_tag.sh" "$LLM_ROOT")
 cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install ${BUILD_TESTS_ARG} -DTRITON_COMMON_REPO_TAG=${TRITON_SHORT_TAG} -DTRITON_CORE_REPO_TAG=${TRITON_SHORT_TAG} -DTRITON_THIRD_PARTY_REPO_TAG=${TRITON_SHORT_TAG} -DTRITON_BACKEND_REPO_TAG=${TRITON_SHORT_TAG} ..
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)